pkg/sql/backfill/backfill.go

```diff
@@ -587,6 +627,17 @@ func (ib *IndexBackfiller) BuildIndexEntriesChunk(
 	// This ought to be chunkSize but in most tests we are actually building smaller
 	// indexes so use a smaller value.
 	const initBufferSize = 1000
+
+	isBytesMonitored := ib.boundAccount.Monitor() != nil
```

Comment:
Would it be possible to always have a monitor here? I think all of these conditionals are a little bit unfortunate.